src/styles/reservation.module.css

```diff
@@ -0,0 +1,111 @@
+html {
+  box-sizing: border-box;
+}
+
+.destination-dropdown {
+  color: white;
+  background-color: yellowgreen;
+}
+
+.wrapper {
+  color: white;
+  background:
+    linear-gradient(rgba(150, 191, 1, 0.8), rgba(150, 191, 1, 0.8)),
+    url("https://images.unsplash.com/photo-1601871925712-3c803d50fa72?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8YmVhY2glMjB2YWNhdGlvbnxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&w=1000&q=80");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  margin: auto auto;
+  padding: 40px;
+  border-radius: 5px;
+  box-shadow: 0 0 10px #000;
+  width: 90%;
```

Comment:
[Optional]
Hi @themonster2015 , 
You can just remove the line18 and 22, so it can look better in mobile view.